utils/xmtpRN/sync.ts

```diff
@@ -146,8 +146,8 @@ const syncClientConversationList = async (account: string) => {
     });
     // Prefetch the conversation list so when we land on the conversation list
     // we have it ready, this will include syncing all groups
-    const conversationList = await prefetchConversationListQuery(account);
-    // TODO: Update notification topics
+    const conversationList = await fetchConversationListQuery(account);
+    subscribeToNotifications({ conversations: conversationList, account });
   } catch (e) {
     logger.error(e, {
       context: `Failed to fetch persisted conversation list for ${account}`,
```

Comment:
would be cool to start handling errors better. Maybe worth waiting until we have a clear strategy tho